src/garage/_functions.py

```diff
@@ -34,10 +34,14 @@ def log_multitask_performance(itr, batch, discount, name_map=None):
             task_name = trajectory.env_infos['task_name'][0]
         except KeyError:
             try:
+                task_id = None
```

Comment:
I think Keren has an alternative change to this logic, which looks a little better.
I just needed the `MetaEvaluator` to work on environments where all tasks are unnamed / continuous.

Reply:
Yeah, when Keren merges PR #1214 I'll remove the logic here.